README.md

```diff
@@ -1,64 +1,88 @@
 # Object Iteration
 
-## Objectives
-- Explain the difference between looping and iteration.
-- Iterate over arrays with the `for...of` statement.
-- Enumerate an object's properties with the `for...in` statement.
+## Problem Statement
+
+After learning about collections (`Array`, `Object`), it's very common to need
+to be able to do some activity for each element in the collection. For each
+evil wizard, disarm them! For each wedding guest, invite them. Yet this is our
+only tool:
 
-## Introduction
-When we create a `for` loop to loop over an array, we base the loop's condition off of the `.length` of the array. This works, but it's a lot of syntactic cruft to remember:
 ```js
 for (let i = 0; i < array.length; i++) {
   // Loop body
 }
 ```
 
-The problem is that we're using a _looping_ construct to perform _iteration_.
+ES2015 introduce a simpler way to express this common path. For `Array`s we now
+have `for...of` and for Objects we now have `for...`in`.
 
-## Looping vs. iteration
-There's a pretty fine line separating the concepts of _looping_ and _iteration_, and only the truly pedantic will call you out if you use one in place of the other.
+## Objectives
 
-Looping is the process of executing a set of statements **repeatedly until a condition is met**. It's great for when we want to do something a specific number of times (`for` loop) or unlimited times until the condition is met (`while` loop).
+1. Define iteration
+2. Discuss what's wrong with `for` loops; "expressivity"
```

Comment:
Should the semicolon be a colon? If it's meant to function as a semicolon, I think it would be better to rephrase to something like "Discuss the problem of expressivity with `for` loops."